test/torchaudio_unittest/librosa_compatibility_test.py

```diff
@@ -180,19 +190,54 @@ def assert_compatibilities(self, n_fft, hop_length, power, n_mels, n_mfcc, sampl
         out_torch = spect_transform(sound).squeeze().cpu()
         self.assertEqual(out_torch, torch.from_numpy(out_librosa), atol=1e-5, rtol=1e-5)
 
-        # test mel spectrogram
+    @parameterized.expand([
+        p._replace(kwargs={'norm': norm, **p.kwargs})
+        for p in [
+            param(n_fft=400, hop_length=200, n_mels=128),
+            param(n_fft=600, hop_length=100, n_mels=128),
+            param(n_fft=200, hop_length=50, n_mels=128),
+        ]
+        for norm in [None, 'slaney']
+    ])
```

Comment:
Use of internal method `_replace` should be avoided.
We use `@parameterized.expand(list(itertools.product([....], [...]))` pattern for this kind of nesting.
Could you try the same?

Reply:
Despite having a leading underscore, I don't believe that `_replace` is an internal method. `parameterized.param` is a `collection.namedtuple`. As per the python docs the leading underscore here is to prevent naming conflicts, not because it is an internal method: "In addition to the methods inherited from tuples, named tuples support three additional methods and two attributes. To prevent conflicts with field names, the method and attribute names start with an underscore." That said, my fingers did shake when I typed it out, so I've replaced it with something else. I don't think that itertools.product would work well here, as it's more suited for trying every combination (the cartesian product) of n_fft, hop_length and n_mels.

Reply:
Your update looks good to me. I was worried that the use of `_replace`, even if it's not internal method, forces contributors to learn the internal of `parameterized.param`, which would raise the bar for writing tests, which could lead to the situation where contributors would not like to contribute anything at all. But now the final form looks good and even better than `itertool` approach I ended up with. Thanks.